src/app/core/model/EventSearchDetails.ts

```diff
@@ -1,6 +1,8 @@
 import {PageableDto} from "./dto/PageableDto";
 
 export interface EventSearchDetails {
+  itemsPerPage: number;
+  page: number;
```

Comment:
Te pola są tu niepotrzebne, przenieśliśmy je do PageableDto

Reply:
Jeśli robiłeś je po to, żeby działał mockService, to lepiej mockService do nowego modelu dostosować niż odwrotnie

Reply:
done